lib/base/serialize.js

```diff
@@ -135,8 +138,8 @@ module.exports.bool = function(input) {
  * @returns {array} new array with transform applied to each element.
  */
 module.exports.map = function(input, transform) {
-  if (_.isArray(input)) {
-    return _.map(input, transform);
+  if (typeof input === 'array' || (typeof input === 'object' && Object.prototype.toString.call(input) === '[object Array]')) {
```

Comment:
if `input` was an array, typeof would be 'object' Could remove `typeof input === 'array'`